components/Cards.vue

```diff
@@ -120,4 +112,34 @@ const popup = [
     //         'A short description about the Hide XP and cFuel Widget',
     // },
 ]
+
+const people = [
```

Comment:
Where do we need people?

Reply:
I was using it as a placeholder, forgot to remove it before pushing.